repository/Seaside-Core.package/WARequest.class/instance/forwarded.st

```diff
@@ -1,3 +1,3 @@
 accessing-headers
 forwarded
-	^ self headerAt: 'forwarded' ifAbsent:[ self headerAt: 'x-forwarded-for' ]
\ No newline at end of file
+	^ self headerAt: 'forwarded'
```

Comment:
Shouldn't we keep the fallback to `x-forwarded-for`?

Reply:
It had a different syntax. I would say we do not support it.

Reply:
Ok. Given that the code has never worked in the first place we can do that. But will that work for you when you upgrade to 3.4 from 3.0? If I'm not mistaken your using this at work, no?

Reply:
Not the ip-based one. I just had to rename that class which unleashed the changes in this PR ;)

Reply:
We use the cookie-based protection filter I added